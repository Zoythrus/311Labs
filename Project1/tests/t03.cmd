 text3_1.txt text3_2.txt 
