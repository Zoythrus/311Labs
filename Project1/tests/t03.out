The greatest gift of life is friendship, and I have received it. Hubert H. Humphrey. 
